h/views/groups.py

```diff
@@ -49,15 +49,18 @@ def edit(self):
     def _js_config(self):
         csrf_token = get_csrf_token(self.request)
 
-        def api_config(route_name: str, method: str, **params):
+        def api_config(route_name: str, method: str, **kw):
```

Comment:
I would normally user `*args` and `**kwargs` in Python but Pyramid's `route_url()` [uses `**kw`](https://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html#pyramid.request.Request.route_url) so changing this to match that.